### PR TITLE
Clarify wording of pack, pack_bits and merkleize

### PR DESCRIPTION
**Pull request by booleanfunction**
This PR has been created on the basis of https://github.com/ethereum/eth2.0-specs/issues/1893 and discussions with @protolambda.

Minor adjustments to clarify the wording of the following SSZ functions (Merkleization section):
- `pack`
- `pack_bits`
- `merkleize`

### CODE REVIEW

**Comment by protolambda** [comment]
Marking it as presentation, not because it's cosmetic, but since it shouldn't change anything for a client that implemented it and passed tests (there are no non-power-of-2 limits in the spec currently), so it's not breaking and can be included in the very next release. Thank you for fixing this :+1:

File: ssz/simple-serialize.md [hunk 1]
```diff
@@ -213,13 +213,17 @@ We first define helper functions:
    * `List[B, N]` and `Vector[B, N]`, where `B` is a basic type: `(N * size_of(B) + 31) // 32` (dividing by chunk size, rounding up)
    * `List[C, N]` and `Vector[C, N]`, where `C` is a composite type: `N`
    * containers: `len(fields)`
-* `pack(value)`: given ordered objects of the same basic type, serialize them, pack them into `BYTES_PER_CHUNK`-byte chunks, right-pad the last chunk with zero bytes, and return the chunks.
-* `pack_bits(bits)`: Given the `bits` of bitlist or bitvector, get `bitfield_bytes` by packing them in bytes and aligning to the start. The length-delimiting bit for bitlists is excluded. And then pack `bitfield_bytes` into `BYTES_PER_CHUNK`-byte chunks, right-pad the last chunk with zero bytes, and return the chunks.
+* `pack(values)`: Given ordered objects of the same basic type:
+   1. Serialize the values into bytes.
+   1. If not aligned to a multiple of `BYTES_PER_CHUNK` bytes, right-pad with zeroes to the next multiple.
+   1. Partition the bytes into `BYTES_PER_CHUNK`-byte chunks.
+   1. Return the chunks.
```
Comment (hwwhww) [inline ssz/simple-serialize.md:220]: nitpicking from a raw markdown reader.
```suggestion
   2. If not aligned to a multiple of `BYTES_PER_CHUNK` bytes, right-pad with zeroes to the next multiple.
   3. Partition the bytes into `BYTES_PER_CHUNK`-byte chunks.
   4. Return the chunks.
```

File: ssz/simple-serialize.md [hunk 2]
```diff
@@ -213,13 +213,17 @@ We first define helper functions:
    * `List[B, N]` and `Vector[B, N]`, where `B` is a basic type: `(N * size_of(B) + 31) // 32` (dividing by chunk size, rounding up)
    * `List[C, N]` and `Vector[C, N]`, where `C` is a composite type: `N`
    * containers: `len(fields)`
-* `pack(value)`: given ordered objects of the same basic type, serialize them, pack them into `BYTES_PER_CHUNK`-byte chunks, right-pad the last chunk with zero bytes, and return the chunks.
-* `pack_bits(bits)`: Given the `bits` of bitlist or bitvector, get `bitfield_bytes` by packing them in bytes and aligning to the start. The length-delimiting bit for bitlists is excluded. And then pack `bitfield_bytes` into `BYTES_PER_CHUNK`-byte chunks, right-pad the last chunk with zero bytes, and return the chunks.
+* `pack(values)`: Given ordered objects of the same basic type:
+   1. Serialize the values into bytes.
```
Comment (hwwhww) [inline ssz/simple-serialize.md:217]:
```suggestion
   1. Serialize `values` into bytes objects.
```

**Comment by booleanfunction** [comment]
@hwwhww thank you for the suggestions, I have updated the commits :)

**Pull request merged**